### DISCUSSION TITLE
Change images according to zoom level?

### DISCUSSION THREAD

**Discussion by gaallo25** [Leaflet/Leaflet#8413] [General]
Good morning, I am trying to make a map with my own images and I do not understand very well how to change the images depending on the zoom level, could you help me?

**Comment by zxh19890103** [comment]
I recommend you reading the source code of https://github.com/Leaflet/Leaflet/blob/main/src/layer/ImageOverlay.js. There must be a update function for every layer to response the zoom/view changes.
